worldedit-core/src/main/java/com/sk89q/worldedit/command/RegionCommands.java

```diff
@@ -291,6 +292,28 @@ public int smooth(Actor actor, EditSession editSession, @Selection Region region
         return affected;
     }
 
+    @Command(
+        name = "/snowsmooth",
+        desc = "Smooth the elevation in the selection with snow layers",
+        descFooter = "Example: '//snowsmooth 1 -m snow_block,snow' would only smooth snow terrain."
+    )
+    @CommandPermissions("worldedit.region.snowsmooth")
+    @Logging(REGION)
+    public int snowSmooth(Actor actor, EditSession editSession, @Selection Region region,
+                          @Arg(desc = "# of iterations to perform", def = "1")
+                              int iterations,
+                          @ArgFlag(name = 'm', desc = "The mask of blocks to use as the height map")
+                              Mask mask,
+                          @ArgFlag(name = 'l', desc = "Set the amount of snow blocks under the snow", def = "1")
+                              int snowBlockCount) throws WorldEditException {
```

Comment:
Can you order these the other way around? I would like to keep the flag ordering the same.